.github/workflows/prerelease.yml

```
@@ -0,0 +1,81 @@
+          9.0.x
+          
+    - name: Check if push to main
+      id: push_check
+      run: echo "is_push_to_main=${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}" >> $GITHUB_OUTPUT
+
+    - name: Restore dependencies
+      run: dotnet restore
+      
+    - name: Build
+      run: dotnet build --no-restore
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)